README.md

```diff
@@ -12,15 +55,3 @@ To run `eslint --fix` on save add to the settings.json file:
   {"language":  "typescriptreact",  "autoFix":  true  }
 ]
```

Comment:
I think you also need the Prettier extension installed for auto format or save https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode

I believe this was my issue at first, because only having that setting did not prevent me from pushing unformatted code.

Reply:
Thanks, I'll add it

Reply:
https://github.com/gnosis/dex-react/tree/add-readme-info#automatically-fixing-code